src/components/LessonCard.js

```diff
@@ -42,17 +46,52 @@ const LessonCard = ({ lesson, onClick }) => {
     }
   };
 
+  const subheader = () => (
+    userTier >= lesson.tier ? (
+      lesson.description
+    ) : (
+      <Grid container direction="row">
+        <Grid item xs={2}>
+          <Lock fontSize="large" />
+        </Grid>
+        <Grid item xs={10}>
+          <Typography variant="subtitle1">
+            <Link href="/user/settings" color="textSecondary" underline="always">
+              <FormattedMessage
+                id="app.lesson_card.blocked1"
+                description="Informs the user that upgrade is required to access"
+                defaultMessage="Upgrade"
+              />
+            </Link>
+            {' '}
+            <FormattedMessage
+              id="app.lesson_card.blocked2"
+              description="Informs the user that upgrade is required to access"
+              defaultMessage="your account to access this lesson."
+            />
+          </Typography>
+        </Grid>
+      </Grid>
+    )
+  );
+
+  const handleOnClick = (e) => {
+    if (userTier >= lesson.tier) {
+      onClick(e);
+    }
```

Comment:
Note for the future: maybe we could do `else: nav to user/settings` so that clicking the leson card anywhere takes you to the upgrade options.